cmake/pml_cc_library.cmake

```diff
@@ -201,4 +204,24 @@ function(pml_cc_library)
     # For example, foo/bar/ library 'bar' would end up as 'foo::bar'.
     add_library(${_PACKAGE_NS} ALIAS ${_NAME})
   endif()
+
+  get_property(INSTALL_TARGETS GLOBAL PROPERTY install_targets_property)
+  if(TARGET ${_NAME} AND NOT ${_NAME} MATCHES ".*openvino.*" AND NOT ${_NAME} IN_LIST INSTALL_TARGETS)
```

Comment:
Hmm, I'm not feeling good about having library-specific goo in this generic `pml_cc_library` rule.

What if we added a whole new flag to `pml_cc_library` which means 'this is a public/external library'. Only in this case would we install headers for that library.

Alternatively, we could just list the headers directly from the packaging so that things are a little less magical and it's explicitly listed someplace exactly what's going into the public package.